### DISCUSSION TITLE
Is it currently possible to load multiple themes as mentioned in the docs?

### DISCUSSION THREAD

**Discussion by Bmorr1123** [MyreMylar/pygame_gui#263] [Q&A] [answered]
Hello I'm using pygame_gui for the first time, and a lot of the stuff I've wanted to do with it has been very easy and seamless, but unfortunately while trying to make a default theme for my different menus, I've encountered an issue.

In the docs it says you can load multiple themes by doing:
```
 manager = pygame_gui.UIManager((800, 600), 'base_theme.json')
 manager.load_theme('menu_theme.json')
 manager.load_theme('hud_theme.json')
```
https://pygame-gui.readthedocs.io/en/latest/theme_guide.html#multiple-theme-files
but when I try this it, pycharm warns me that manager.load_theme is an unknown method.
When I tried to just ignore this error incase it was just a pycharm quirk, I got this error:
```
AttributeError: 'UIManager' object has no attribute 'load_theme'
```

So my question is, is this actually implemented or is it in future plans for the library? I see that it's currently on version 0.6.4, so I can understand some stuff not being complete yet.

**Comment by MyreMylar** [comment, marked as answer]
Hey there, looks like there was a little error in the docs. It should read:

```
 manager = pygame_gui.UIManager((800, 600), 'base_theme.json')
 manager.get_theme().load_theme('menu_theme.json')
 manager.get_theme().load_theme('hud_theme.json')
```

To work correctly, which reads a little oddly I suppose but is how the code works right now.
